loader/loader.go

```diff
@@ -358,6 +358,8 @@ type Loader struct {
 
 	// for every worker goroutine, not for every data file
 	workerWg *sync.WaitGroup
+	// for other goroutines
+	wg *sync.WaitGroup
```

Comment:
I prefer to use `sync.WaitGroup` rather than `*sync.WaitGroup` for single `Loader` instance (then we can remove L392, do not need to keep uniform for old ugly code)

Reply:
addressed in https://github.com/pingcap/dm/pull/578/commits/02afc7ebfad754598a6266bdcc1acd1056f6c3a3